e2e/ansible/roles/k8s-gke-detach/tasks/detach-delete.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Getting the total number of disk in a nodes
+  shell: gcloud compute disks list --filter="NAME:( {{ outer_item }} )" | grep disk | awk '{print $1}'
+  args:
+    executable: /bin/bash
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+  register: disk
+
```

Comment:
@nsathyaseelan How are we getting the list of disks attached? Are we getting disks list by finding the node name along with disk name?

Reply:
@dargasudarshan we are getting disk list by finding the node name.